src/libs/actions/IOU.js

```diff
@@ -303,7 +304,7 @@ function createSplitsAndOnyxData(participants, currentUserLogin, amount, comment
             onyxMethod: CONST.ONYX.METHOD.MERGE,
             key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${groupChatReport.reportID}`,
             value: {
-                0: {pendingAction: null},
+                ...(existingGroupChatReport ? {} : {[groupCreatedReportAction.reportActionID]: {pendingAction: null}}),
                 [groupIOUReportAction.sequenceNumber]: {pendingAction: null},
```

Comment:
Shouldn't this use `reportActionID` instead of `sequenceNumber`?

Reply:
Good catch

Reply:
Fixed